src/app/routes/home/getInitialData/index.js

```diff
@@ -14,13 +18,46 @@ const transformJson = pipe(
   filterGroupsWithoutStraplines,
 );
 
-export default async path => {
-  const { json, ...rest } = await fetchPageData(path);
+export default async (path, service) => {
+  const { SIMORGH_APP_ENV, SIMORGH_BASE_URL } = process.env;
+  const config = serviceConfigs[service];
+
+  const hasRadioSchedule = pathOr(
+    false,
+    ['default', 'radioSchedule', 'hasRadioSchedule'],
+    config,
+  );
+
+  if (!hasRadioSchedule) {
+    const { json, ...rest } = await fetchPageData(path);
+
+    return {
+      ...rest,
+      ...(json && {
+        pageData: transformJson(json),
+      }),
+    };
+  }
+
+  const radioSchedulesUrl = getRadioScheduleEndpoint({
+    baseUrl: SIMORGH_BASE_URL,
+    service,
+    env: SIMORGH_APP_ENV,
+    queryString: getQueryString(path),
+  });
+
+  const [{ json, ...rest }, radioSchedulesData] = await Promise.all([
+    fetchPageData(path),
+    fetchPageData(radioSchedulesUrl, true),
+  ]);
 
   return {
     ...rest,
     ...(json && {
-      pageData: transformJson(json),
+      pageData: {
+        ...transformJson(json),
+        radioSchedules: radioSchedulesData && radioSchedulesData.json,
```

Comment:
Could have this as a preprocess function instead

Reply:
I also had another PR where this was passed down separately and added to a context provider, but this is more straightforward to begin with.

Reply:
Also, to reduce the amount of JSON we add to the page, we could run this through the processing function here. Would need to do be able to do that in the `useData` hook too though.